homeassistant/components/watergate/strings.json

```diff
@@ -19,6 +19,42 @@
     }
   },
   "entity": {
+    "event": {
+      "auto_shut_off_volume": {
+        "name": "Volume Auto shut-off",
+        "state_attributes": {
+          "event_type": {
+            "state": {
+              "volume_threshold": "Volume",
+              "duration_threshold": "Duration"
+            }
+          },
+          "volume": {
+            "name": "Volume"
+          },
+          "duration": {
+            "name": "Duration"
+          }
```

Comment:
I think its fine keeping them (as _maybe_ they are used in the future)


When typing this, it dawned on me, maybe it was used in the automation editor:
![image](https://github.com/user-attachments/assets/20a75207-6cc6-43aa-a7d7-29de4388f871)

Reply:
But please use reference keys for recurring names: `[%key:component::watergate::entity::........%]`